server/pbenchinacan/load_keycloak.sh

```diff
@@ -44,15 +76,20 @@ echo
 echo "Creating ${REALM} realm"
 echo "--------------"
 
-curl -i -X POST "${KEYCLOAK_HOST_PORT}/admin/realms" \
+status_code=$(curl -f -i -o /dev/null -w "%{http_code}" -X POST "${KEYCLOAK_HOST_PORT}/admin/realms" \
   -H "Authorization: Bearer ${ADMIN_TOKEN}" \
   -H "Content-Type: application/json" \
-  -d '{"realm": "'${REALM}'", "enabled": true}'
+  -d '{"realm": "'${REALM}'", "enabled": true}')
+
+if [[ "${status_code}" != "201" ]]; then
+  echo ${status_code}
```

Comment:
Since we're going to the trouble of invoking `echo`, we might as well add some text to provide context (e.g., what operation failed with the provided status).

Ditto line 150 and, possibly, 120?